mindjunkies/courses/management/commands/populate_category.py

```
@@ -0,0 +1,48 @@
+    help = 'Populate the database with sample CourseCategory data with related categories'
+
+    def handle(self, *args, **options):
+        self.stdout.write("Starting course category data population...")
+
+        # Clear existing course category data
+        CourseCategory.objects.all().delete()
+
+        # Create course categories with related groupings
+        categories = [
+            {"name": "Data Science", "description": "Master data analysis, visualization, and statistical modeling."},
+            {"name": "Machine Learning", "description": "Dive into algorithms and predictive modeling for data science."},
```

Ruff (0.8.2)

Line too long (122 > 119)

(E501)